fabric-rendering-v1/src/main/java/net/fabricmc/fabric/api/client/rendering/v1/BuiltinItemRendererRegistry.java

```diff
@@ -0,0 +1,26 @@
+package net.fabricmc.fabric.api.client.rendering.v1;
+
+import net.minecraft.item.Item;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.fabric.impl.client.rendering.BuiltinItemRendererRegistryImpl;
+
+/**
+ * This registry holds {@linkplain BuiltinItemRenderer builtin item renderers} for items.
+ */
+@Environment(EnvType.CLIENT)
+public interface BuiltinItemRendererRegistry {
+	BuiltinItemRendererRegistry INSTANCE = BuiltinItemRendererRegistryImpl.INSTANCE;
```

Comment:
Maybe note that this is how to get a renderer (some people have asked such questions on discord before)

Reply:
It's not how to get a renderer, though; that's not exposed in the API.

Reply:
Oh, I mean how to get a renderer registry :sweat_smile: